src/symbanded/bandedeigen.jl

```diff
@@ -1,3 +1,34 @@
+## eigvals routine
```

Comment:
The `eigvals` methods are moved to `bandedeigen.jl` from `symbanded.jl` so that they're in the same file as `eigen` and are easy to discover.